nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -1272,7 +1330,7 @@ impl BundleCollection {
                 "Could not contact sled",
             )
             .await?;
-            return Ok(CollectionStepOutput::None);
+            bail!("Could not contact sled");
```

Comment:
i realize that this was already not the case, but it kinda feels like the underlying error from the client ought to be formatted here as well?